app/services/get-export-csv.js

```diff
@@ -168,5 +155,5 @@ var parseTotalSummaryTable = function (totalSummary) {
 }
 
 var formatCapacityValue = function (capacity) {
-  return parseFloat(capacity).toFixed(2) + '%'
+  return parseInt(capacity) + '%'
```

Comment:
I thought the requirement was that we show two decimal places?

Reply:
Yeah you're right, I originally wrote that function but when I was getting feedback from Urban about these new export files he seemed to change his mind and asked for it to be a whole number. Below is his feedback from email:

`As far as I can see, the data displayed in the mock exports meets the requirement. I have copied in Andy who raised this point. In line with other feedback provided could the capacity percentage be displayed as whole number percentage, i.e. 97% instead 96.72%.`

Looking back though, I should probably be using Math.round rather than parseInt

Reply:
👍